docs/traffic-policy/variables/endpoint.mdx

```diff
@@ -0,0 +1,73 @@
+import ConfigExample from "../../../src/components/ConfigExample.tsx";
```

Comment:
Instead of breaking these up by namespace I was thinking more by "phase" (as in on_tcp_connect, on_http_request, on_http_response). What do you think?

Reply:
I was thinking namespace to be able to have different reference tables for tcp vs tls endpoints (since conn.tls vars isn't useful for tcp). Open to building up by phase too, might be missing how you'd want to use them

Reply:
nah you're right.... tls has different vars than tcp